Source/Initialization/WarpXAMReXInit.H

```diff
@@ -9,6 +9,10 @@
 
 #include <AMReX.H>
 
+#ifdef AMREX_USE_MPI
+#   include <mpi.h>
+#endif
```

Comment:
Since `MPI_Comm` us used unconditionally below, this include would be better:
```suggestion
#ifdef AMREX_USE_MPI
#   include <AMReX_ccse-mpi.H>
#endif
```

It is also pulled by `<AMReX.H>`.

Reply:
ah ! Maybe by doing this we could avoid including AMReX.H in that file